tools/scripts/stm32.mk

```diff
@@ -163,8 +163,8 @@ post_build: $(HEX)
 	$(MUTE) $(foreach inc, $(EXTRA_INC_PATHS), sed -i '/Core\/Inc"\/>/a <listOptionValue builtIn="false" value="$(inc)"\/>' $(PROJECT_BUILDROOT)/.cproject;) $(HIDE)
 	$(MUTE) $(foreach flag, $(CPROJECTFLAGS), sed -i '/USE_HAL_DRIVER"\/>/a <listOptionValue builtIn="false" value="$(flag)"\/>' $(PROJECT_BUILDROOT)/.cproject;) $(HIDE)
 
-PHONY += $(PLATFORM)_project_build
-$(PLATFORM)_project_build: $(PROJECT_TARGET) $(LIB_TARGETS)
+PHONY += $(PLATFORM)_sdkbuild
```

Comment:
commit title suggests this is for Xilinx only but you have a stm32 related change here.

Reply:
Actually I think you should rather do each rule like you did it in the following commit:
`tools: scripts: *.mk: sdkopen rule `
Put the top-level generic.mk rule altogether with all platform specific rules that it uses in the same commit. That way, when somebody checks-out this commit, the rule will work for any platform.